quinn-proto/src/crypto.rs

```diff
@@ -87,6 +87,11 @@ pub trait ClientConfig<S>
 where
     S: Session,
 {
+    /// Construct the default configuration
```

Comment:
Should we use `Default` for this, instead?

Reply:
We can't implement `Default` on `Arc<rustls::Client/ServerConfig>`.